### ISSUE TITLE
Profile isn't listed, if it contains a dot in the profilename (or maybe other chars)

### ISSUE THREAD

**Issue by michaelfecher** [Systanics/CloudGlance#12]
# Some background details
I'm using the [aws-sso-util tool](https://github.com/benkehoe/aws-sso-util) to manage all my SSO profiles.
There's a neat command, called `populate` which automatically reads all profiles from a specified SSO start URL and stores them in the AWS config.

The AWS config for an autopopulated profile looks like the following:
```
[profile amazon-aws-cool-account-dev.AWSAdministratorAccess]
sso_start_url = https://foobar.awsapps.com/start
sso_region = eu-west-1
sso_account_name = amazon-aws-cool-account-dev
sso_account_id = 12345
sso_role_name = AWSAdministratorAccess
region = eu-west-1
etc.
```

# Bug description
The bug happens, when I want to add a profile (e.g. amazon-aws-cool-account-dev.AWSAdministratorAccess).
It isn't listed at all.
So I only can add those profiles, if I manually rename all of them (awww!).

Guess it's about the dot (.)
If I manually rename the profile name in my AWS config, it's working out.

**Comment by rehanvdm** [comment]
Yeah, I certainly have no test cases that covers names with a dot in them. Should be able to recreate and fix in the next release

**Comment by rehanvdm** [comment]
Turns out that some INI parsers parse a dot in the name as a path indicator used for nesting. The INI parser we are using https://www.npmjs.com/package/ini (30 million downloads) seem to this as well, https://github.com/npm/ini/issues/22. I did a workaround knowing and it seems to be working, even with more dots and deeply nested objects.

![image](https://user-images.githubusercontent.com/22810856/193737119-a446e161-9648-449e-8cb5-f9db699c384d.png)

Fixed, will be in the next release.
